src/requirements/data/majors/comm.ts

```diff
@@ -1,33 +1,26 @@
 import { CollegeOrMajorRequirement } from '../../types';
-import { includesWithSingleRequirement, includesWithSubRequirements } from '../checkers-common';
+import { includesWithSingleRequirement } from '../checkers-common';
 
 const commRequirements: readonly CollegeOrMajorRequirement[] = [
   {
     name: 'Core Courses',
     description: '15 credits in COMM 1101, COMM 1300, COMM 2010, COMM 2310, COMM 2820',
     source: 'https://communication.cals.cornell.edu/undergraduate-program/major-requirements/',
-    checker: includesWithSubRequirements(
-      ['COMM 1101'],
-      ['COMM 1300'],
-      ['COMM 2010'],
-      ['COMM 2310'],
-      ['COMM 2820']
+    checker: includesWithSingleRequirement(
+      'COMM 1101',
+      'COMM 1300',
+      'COMM 2010',
+      'COMM 2310',
+      'COMM 2820'
     ),
-    subRequirementProgress: 'every-course-needed',
     fulfilledBy: 'credits',
     minCount: 15,
   },
```

Comment:
I wonder if this could be `fulfilledBy: 'courses'`. Since it seems you would have to take every course they list.

Reply:
@tcho6319 I updated it, but I checked the website again and the requirement seems to change already. We need to make a pass over all requirements data before launch.